spec/requests/api/v1/applications_spec.rb

```diff
@@ -46,9 +46,15 @@ def create_resource(**attrs)
   end
 
   describe("accept") do
```

Comment:
Can you update the describe name to `POST /api/v1/npq-applications/:ecf_id/accept`, to follow the naming format.

Reply:
same for v2, v3